students/david_baylor/lesson3/strformat_lab.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+from decimal import Decimal
+
+
+"""
+strformat_lab.py
+By David Baylor on 10/29/19
+uses python 3
+
+"""
+
+def formatter(in_tuple):
+    form_string = "the {} numbers are: {}".format(len(in_tuple), "{}"+", {}"*(len(in_tuple)-1))
+
+    return form_string.format(*in_tuple)
+
+def tableMaker(row):
```

Comment:
Function names (as well as almost everything else) should be snake_case in Python: table_maker(row):